core/src/test/java/nl/nn/adapterframework/http/rest/OpenApiTestBase.java

```diff
@@ -109,9 +109,11 @@ private static TaskExecutor getTaskExecutor() {
 		return taskExecutor;
 	}
 
-	protected HttpServletRequest createRequest(String method, String uri) {
+	protected MockHttpServletRequest createRequest(String method, String uri) {
 		MockHttpServletRequest request = new MockHttpServletRequest(method.toUpperCase(), uri);
 		request.setServerName("dummy");
+		request.setServletPath("/api");
+		request.setRequestURI("/api"+uri);
```

Comment:
Ik durf het bijna niet te zeggen, maar ik denk dat het verstandig is om hier zowel servletPath als contextPath op iets herkenbaars te zetten, en misschien ook nog wel variabel te maken, om te zien dat die elementen op de juiste plek in het gegenereerde schema terugkomen.